modules/prometheus/src/main/scala/com/evolutiongaming/skafka/consumer/PrometheusConsumerMetrics.scala

```diff
@@ -12,7 +13,10 @@ object PrometheusConsumerMetrics {
     val Default: Prefix = "skafka_consumer"
   }
 
-  def apply[K, V](registry: CollectorRegistry, prefix: Prefix = Prefix.Default): ClientId => Consumer.Metrics = {
+  def apply[F[_] : Async, K, V](
+    registry: CollectorRegistry,
+    prefix: Prefix = Prefix.Default)
+    (clientId: ClientId): Metrics[F] = {
```

Comment:
as I already said - this won't work with prometheus this way.

Reply:
Forgot it was two of them, reverted.